src/js/components/advertisement/advertisement-consumer-settlement-ledger-view.jsx

```diff
@@ -57,9 +69,14 @@ class AdvertisementConsumerSettlementLedgerView extends Component {
         });
     }
 
+    redirectToAd (data) {
```

Comment:
it is not redirect and you should not use shortcuts. rename to `openAdvertisementLink`